msteams-platform/concepts/build-and-test/apps-localization.md

```diff
@@ -64,26 +68,29 @@ You will then want to add the 'localizationInfo' property with the default langu
 }
 ```
 
-You can provide additional .json files with translations of all the user facing strings in your manifest. These files must adhere to the [Localization file JSON schema](../../resources/schema/localization-schema.md) and they must be added to the 'localizationInfo' property of your manifest. Each file correlates to a language tag which the Teams client uses to choose the appropriate strings. The language tag takes the form of <language>-<region> but it is recommended to omit the <region> portion to target all regions that support the desired language.
+You can provide additional .json files with translations of all the user facing strings in your manifest. These files must adhere to the [Localization file JSON schema](../../resources/schema/localization-schema.md) and they must be added to the `localizationInfo` property of your manifest. Each file correlates to a language tag which the Teams client uses to select the appropriate strings. The language tag takes the form of `<language>-<region>` but it is recommended to omit the `<region>` portion to target all regions that support the desired language.
```

Comment:
"it is recommended to omit" --> should this be "you may omit"?
It's not always appropriate to remove the region, where there are large differences between say pt-pt vs pt-br, or zh-cn vs zh-tw.